cmd/manager/main.go

```diff
@@ -157,13 +158,26 @@ func main() {
 		setupLog.Error(err, "unable to get ingress config.")
 		os.Exit(1)
 	}
-	if deployConfig.DefaultDeploymentMode == string(constants.Serverless) {
+
+	ksvcFound, ksvcCheckErr := utils.IsCrdAvailable(cfg, knservingv1.SchemeGroupVersion.String(), constants.KnativeServiceKind)
+	if ksvcCheckErr != nil {
+		setupLog.Error(ksvcCheckErr, "error when checking if Knative Services are available")
```

Comment:
Minor: seems like not all changes are synced from https://github.com/opendatahub-io/kserve/pull/265?

Reply:
Fixed